go.mod

```diff
@@ -1,8 +1,10 @@
 module github.com/fullstorydev/grpcurl
 
+go 1.14
```

Comment:
You need to remove this in order for it to work with Go v1.11.

Reply:
Sorry. Auto-added by `go mod tidy`. I suppose that's exactly how `protobuf` ended up with this erroneous dependency.